content/java/concepts/calendar/terms/getCalendarType/getCalendarType.md

```diff
@@ -0,0 +1,55 @@
+---
+Title: '.getCalendarType()'
+Description: 'Returns the calendar type of the Calendar instance'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Calendar'
+  - 'Methods'
+CatalogContent:
+  - 'learn-java'
+  - 'paths/computer-science'
+---
+
+The `.getCalendarType()` method of the `Calendar` class is used to return a String type of the calendar used.
+
+## Syntax
+
+```pseudo
+
+public String getCalendarType()
+
+```
+
```

Comment:
Generally, there should be a sentence here that explains the syntax. In some cases it will be minimal, but it should address the parameters(if there are any); special cases; return values or any other notes.